types/lodash/common/function.d.ts

```diff
@@ -655,7 +655,6 @@ declare module "../index" {
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, plc2: __, arg3: T3): Function3<T1, T2, T4, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, arg1: T1, plc2: __, arg3: T3): Function2<T2, T4, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, arg2: T2, arg3: T3): Function2<T1, T4, R>;
-        <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, arg1: T1, arg2: T2, arg3: T3): Function1<T4, R>;
```

Comment:
This seems to be working as intended. I don't think it should be removed.

Reply:
Really?
It looks not working on `pnpm test`.

```diff
❯ git --no-pager diff --cached
diff --git a/types/lodash/common/function.d.ts b/types/lodash/common/function.d.ts
index 46ac3b3746..7628278464 100644
--- a/types/lodash/common/function.d.ts
+++ b/types/lodash/common/function.d.ts
@@ -655,6 +655,7 @@ declare module "../index" {
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, plc2: __, arg3: T3): Function3<T1, T2, T4, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, arg1: T1, plc2: __, arg3: T3): Function2<T2, T4, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, arg2: T2, arg3: T3): Function2<T1, T4, R>;
+        <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, arg1: T1, arg2: T2, arg3: T3): Function1<T4, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, plc2: __, plc3: __, arg4: T4): Function3<T1, T2, T3, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, arg1: T1, plc2: __, plc3: __, arg4: T4): Function2<T2, T3, R>;
         <T1, T2, T3, T4, R>(func: Function4<T1, T2, T3, T4, R>, plc1: __, arg2: T2, plc3: __, arg4: T4): Function2<T1, T3, R>;
```

```
❯ pnpm -w run test lodash

> definitely-typed@0.0.3 test /Users/yusei/Workspace/repos/github.com/DefinitelyTyped/DefinitelyTyped
> node --enable-source-maps node_modules/@definitelytyped/dtslint/ types "lodash"

dtslint@0.2.20
Error:
/Users/yusei/Workspace/repos/github.com/DefinitelyTyped/DefinitelyTyped/types/lodash/lodash-tests.ts
  7304:5  error  TypeScript@4.7, 4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4, 5.5 expected type to be:
  () => number
got:
  Function1<unknown, number>  @definitelytyped/expect

✖ 1 problem (1 error, 0 warnings)

    at combineErrorsAndWarnings (/Users/yusei/Workspace/repos/github.com/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/@definitelytyped+dtslint@0.2.20_typescript@5.5.0-dev.20240509/node_modules/@definitelytyped/dtslint/src/index.ts:259:26)
    at runTests (/Users/yusei/Workspace/repos/github.com/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/@definitelytyped+dtslint@0.2.20_typescript@5.5.0-dev.20240509/node_modules/@definitelytyped/dtslint/src/index.ts:250:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at main (/Users/yusei/Workspace/repos/github.com/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/@definitelytyped+dtslint@0.2.20_typescript@5.5.0-dev.20240509/node_modules/@definitelytyped/dtslint/src/index.ts:103:22)
 ELIFECYCLE  Test failed. See above for more details.
```

Reply:
And this pattern is followed here so I delete it.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d293cb430e2778a9f2446ddd07e7f6eb8f967cba/types/lodash/common/function.d.ts#L669

Reply:
There is not working.
https://stackblitz.com/edit/fjjtuv?file=index.ts

May be your stackblitz dependency is too old.

Reply:
Ah, yeah I missed the overload below which does this. Thanks!